providers/nfs_provider/deploy/nfs-pv.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 The SODA Authors.
```

Comment:
Can we arrange all nfs server deployment things together to one yaml so that that can be setup first and then install kahu

Reply:
Done